components/FaceScanner/FaceScanner.tsx

```diff
@@ -100,27 +103,16 @@ export const FaceScanner: React.FC<FaceScannerProps> = props => {
   }
 
   async function handleFacesDetected({faces}) {
-    checkBlink(faces[0]);
-
-    if (counter == MAX_COUNTER) {
-      setCounter(counter + 1);
-      cameraRef.pausePreview();
-
-      setScreenColor('#ffffff');
-      setInfoText(t('faceProcessingInfo'));
-
-      const result = await cropEyeAreaFromFace(
-        picArray,
-        props.vcImage,
-        faceToCompare,
-      );
-      return result ? props.onValid() : props.onInvalid();
-    } else if (faces.length > 0) {
+    if (counter < MAX_COUNTER) {
```

Comment:
Where we are incrementing the counter?

Reply:
We are incrementing it in line no 126 @sree96. If the captured frame is satisfying the conditions we will consider it as valid frame and then increment it counter.

we are capturing only 15 frames in liveness detection flow so when we are capturing the 15th frame counter will be 14 and the if condition will be satisfied and if it is valid we will increment the counter to 15 and after capturing 15 it will go into else part and in else part we are freezing the camera so there is no use in incrementing the counter here so i have removed it from else part.